_dashboards/visualize/maps.md

```diff
@@ -120,6 +120,58 @@ To show a subset of the data in the index, filter the data. The following exampl
 For large datasets, you may want to avoid loading data for the whole map. To load data only for the part of the map that is currently visible, select the **Only request data around map extent** checkbox.
```

Comment:
```suggestion
For large datasets, you may want to avoid loading data for the full map. To load data only for specific a geographic area, select **Only request data around map extent**.
```

Reply:
@vagimeli This is actually part of the previous selection for overall filtering and was accidentally selected, but not technically part of this PR. I can still include this change because it's so minor, though.